controllers/topics/index.js

```diff
@@ -4,7 +4,8 @@ const {
   followMainFeedTopic,
   unfollowMainFeedTopic,
   addTopicToChatTab,
-  removeTopicFromChatTab
+  removeTopicFromChatTab,
+  createToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `createToken` is added correctly. No issues found.
```diff
+   createToken
 } = require('../../services/getstream');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->